### PR TITLE
Use dirty tricks to minify better

### PR DESCRIPTION
**Pull request by subzey**
Dirty tricks and questionable techniques that shrinks .min.gz down to 259 bytes

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -112,16 +114,16 @@ NanoEvents.prototype = {
...
+    if (!list || !list[0]) return false
 
-    var copy = list.slice(0)
+    list = list.slice()
 
-    var args = [].slice.call(arguments, 1)
-
-    for (var i = 0; i < copy.length; i++) {
-      var l = copy[i]
+    for (
```
Comment (ai) [inline index.js]: This trick is too tricky :D

Comment (subzey) [reply index.js]: Feel free to cherry pick changes you like and drop the ones you don't like

**Comment by ai** [comment]
Great! Can you just revert `for` tricks (it is too tricky) =^_^=

File: index.js (hunk 1 above)
Comment (ai) [reply index.js]: Sure! I will revert it.

**Comment by subzey** [comment]
Is `for (var i = 0; list[i]; i++) {` too tricky?

**Comment by subzey** [comment]
Force-pushed revert of for blak magik
Added since last time:
- check `list[i]` directly, instead of `.length` comparison
- change signature of `add()` so `.once()` implementation looks more like `.on()`: gzip loves duplicate runs of text

**Comment by ai** [comment]
Nope :) I missed it on revert. Will add it back right now.

**Comment by subzey** [comment]
OMG, sorry for the force-push, I didn't see you're doing something with this commit

**Comment by ai** [comment]
@subzey oops :D maybe it is my fail. Can you check current `master`? Did I copy all your ideas (I committed them with your name).

**Comment by subzey** [comment]
It all looks just fine!

**Pull request merged**